### PR TITLE
Tag AWS* and deps

### PR DESCRIPTION
**Pull request by samoconnor**
AWSSNS - add SMS sending
Misc v0.6 tweaks.
Fix https://github.com/samoconnor/SymDict.jl/issues/2

### CODE REVIEW

File: AWSSQS/versions/0.1.4/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.5
+AWSCore 0.1
+SymDict
```
Comment (tkelman) [inline AWSSQS/versions/0.1.4/requires]: looks like this needs 0.1.2 for the lowercase function name

**Comment by tkelman** [comment]
Evidently the git tags don't match the sha values any more

**Comment by samoconnor** [comment]
@tkelman, do you understand why travis is failing now?

**Comment by tkelman** [comment]
You're not allowed to modify the sha of existing tags.

**Comment by samoconnor** [comment]
got it, fixed

**Pull request merged**